### DISCUSSION TITLE
what's for to add meta_dict when invoking LoadImaged ?

### DISCUSSION THREAD

**Discussion by luxunxiansheng** [Project-MONAI/MONAI#6972] [General]
Hi, there,  
        Once invoking Loadimaged ,  besides the original metatensor item,  one more dict item  "postfix_meta_dict"  is appended to the orignal dict data sturcutre .   what's the purpose ?  when will that meta_dict be used?  should I care it in my code ?

thanks

**Comment by wyli** [comment]
Hi, that was developed before the metatensor API, it exists only for backward compatibility for monai <=0.9, `Loadimaged(image_only=True, ...)` will remove the additional key.

**Reply by luxunxiansheng** [reply]
One more question:  The meta info will be used in the transformations which follow the LoadImaged ?  or only the affine is used in the following transformations?
